### PR TITLE
ASB role should handle user's funky passwords

### PR DESCRIPTION
**Pull request by djzager**
Currently, when the user provides a dockerhub_user_pass containing
special characters (ie. myawesome;$password) the rendering of the
template will fail. Quoting the important values should prevent special
characters from being evaluated.

- Update asb main task to handle user/pass special characters
- Update to not include jinja2 templating delimiters in when statement

### CODE REVIEW

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@
   - name: Create a new project for the {{ asb_project }}
     shell: "{{ oc_cmd }} new-project {{ asb_project }}"
     register: new_asb_project
-    when: project.stdout.find( "{{ asb_project }}" ) == -1
+    when: project.stdout.find( asb_project ) == -1
```
Comment (jmrodri) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: this looks weird to me.

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml [hunk 2]
```diff
@@ -43,9 +43,9 @@
         -p REGISTRY_TYPE={{ broker_registry_type }}
         -p REGISTRY_URL={{ broker_registry_url }}
         -p DEV_BROKER={{ broker_dev_broker }}
-        -p DOCKERHUB_ORG={{ dockerhub_org_name }}
-        -p DOCKERHUB_PASS={{ dockerhub_user_password }}
-        -p DOCKERHUB_USER={{ dockerhub_user_name }}
+        -p DOCKERHUB_ORG='{{ dockerhub_org_name }}'
+        -p DOCKERHUB_PASS='{{ dockerhub_user_password }}'
+        -p DOCKERHUB_USER='{{ dockerhub_user_name }}'
```
Comment (jmrodri) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: 👍

File: ansible/roles/ansible_service_broker_setup/tasks/main.yml (hunk 1 above)
Comment (djzager) [inline ansible/roles/ansible_service_broker_setup/tasks/main.yml]: What you should see, before the change:
```
TASK [ansible_service_broker_setup : Create a new project for the ansible-service-broker] *************************************************************************************************************************
 [WARNING]: when statements should not include jinja2 templating delimiters such as {{ }} or {% %}. Found: project.stdout.find( "{{ asb_project }}" ) == -1

changed: [localhost]
```

After:
```
TASK [ansible_service_broker_setup : Create a new project for the ansible-service-broker] *************************************************************************************************************************
changed: [localhost]
```

**Comment by shawn-hurley** [comment]
ACK

**Pull request merged**